### PR TITLE
Convert RST to markdown

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+in a Python 3.6 environments.
+You should also add tests for any new features and bug fixes.
+
+quipucords uses [pytest](http://pytest.org/) for testing.
+
+
+Running all tests:
+******************
+
+.. code-block:: bash
```
Comment (myersCody) [inline CONTRIBUTING.md]: Do we need this? It looks like leftover `rst`? Showing up awkward in view file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+## Unit Testing
+To run the unit tests, use the following command::
+```
+make test
+```
+
+# <a name="issues"></a> Issues
+To report bugs for qpc [open issues](https://github.com/quipucords/qpc/issues>) against this repository in Github. Complete the issue template when opening a new bug to improve investigation and resolution time.
```
Comment (myersCody) [inline README.md]: this link is broken 😬

File: CONTRIBUTING.md (hunk 1 above)
Comment (kholdaway) [reply CONTRIBUTING.md]: fixed it

File: README.md (hunk 2 above)
Comment (kholdaway) [reply README.md]: Fixed

**Pull request merged**